command/rm.go

```diff
@@ -77,6 +78,12 @@ func (d Delete) Run(ctx context.Context) error {
 		printError(d.fullCommand, d.op, err)
 		return err
 	}
+	if !hasSameBuckets(srcurls) {
+		err := errors.New("one rm command cannot be used for object removal of more than one bucket")
```

Comment:
```suggestion
		err := fmt.Errorf("one rm command cannot be used for object removal of more than one bucket")
```

We did not use `errors` package before. For consistency purposes, let's use `fmt.Errorf`.